daemon/user.go

```diff
@@ -143,7 +143,7 @@ func (d *daemon) DestroyUser(ctx context.Context, request *pb.DestroyUserRequest
 
 	if err := d.users.DeleteUserByUsername(request.Username); err != nil {
 		log.Error().Msgf("User delete error %v", err)
-		return &pb.DestroyUserResponse{Message: fmt.Sprintf("Error on deleting user %s, error: %v", request.Username, err)}, nil
+		return &pb.DestroyUserResponse{Message: fmt.Sprintf("Error on deleting user %s, error: %v", request.Username, err)}, err
```

Comment:
What is the reason of changing this ?

Reply:
Because we have to send back an error to the client since we got an error if its inside that `if` statement